src/Model/Filter/Boolean.php

```diff
@@ -25,7 +25,12 @@ public function process()
             return;
         }
 
-        $value = strtolower($this->value());
+        $value = $this->value();
+        if (!is_scalar($value)) {
+            return;
+        }
+
+        $value = strtolower($value);
```

Comment:
I'll blame it on whoever forgot to add tests for actual boolean values :stuck_out_tongue: I can get that fixed up too, the `is_string()` test should do it.